core/dbt/parser/models.py

```diff
@@ -22,8 +22,9 @@ def get_compiled_path(cls, block: FileBlock):
         return block.path.relative_path
 
     def render_update(
-        self, node: IntermediateNode, config: ContextConfig
+        self, node: ParsedModelNode, config: ContextConfig
     ) -> None:
+        node.meta["is_statically_extractable"] = True
```

Comment:
I don't think we can use `meta` for this. That's a property that users can set in yaml files ([docs](https://docs.getdbt.com/reference/resource-properties/meta)). If the user sets any yaml properties for the node, even if they don't set any `meta` properties, the node drops `{'is_statically_extractable': True}` during yaml parsing [here](https://github.com/fishtown-analytics/dbt/pull/3495/files#diff-a25a77362db78f4ddc0675bcbd4775e158631076c9918ce5b64cfcd9f6db6bceR268). The upshot is that we significantly undercount the number of statically extractable + extracted paths.

Is there a different preexisting, internal-use node property we could set here? Or another way to tag the node (without adding having to adjust the manifest schema)?

I'd also be open to adjusting the `v2` manifest schema so that, instead of just adding a standalone `created_at` property, we added a `parsing` dictionary (which could include `created_at`, `statically_extractable`, `statically_extracted`, total parse time, etc).

Reply:
One option might be to store the information in an attribute/dictionary that is filtered out in serialization so that we don't need to update the manifest schema. I couldn't do that with 'created_at' because it had to be in the partial_parse.msgpack file, but I think the information we're discussion here doesn't need to be stored.

I suspect that we're going to keep running into this "new attribute vs updating schema" issue, so we might want to think about something like the 'parsing' dictionary Jeremy proposes, maybe 'internal' or something, where we can put data that probably is not relevant except internally.

Reply:
Oh, I see Kyle already is filtering out that info. Nevermind :)